pkg/crc/api/handlers.go

```diff
@@ -215,8 +215,10 @@ func (h *Handler) UnsetConfig(c *context) error {
 
 func (h *Handler) GetConfig(c *context) error {
 	var req client.GetOrUnsetConfigRequest
-	if err := c.Bind(&req); err != nil {
-		return err
+	if len(c.requestBody) != 0 {
+		if err := c.Bind(&req); err != nil {
```

Comment:
See the conversation in https://github.com/code-ready/crc/pull/2631

Reply:
In that case it was working before, if we pass `-d "{}"` in curl command we are getting all the configs